OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -24,16 +20,22 @@ public class ModInputHandler : IModInputHandler
         private readonly Dictionary<long, IModInputCombination> _comboRegistry = new Dictionary<long, IModInputCombination>();
         private readonly HashSet<InputCommand> _gameBindingRegistry = new HashSet<InputCommand>();
         private readonly HashSet<IModInputCombination> _toResetOnNextFrame = new HashSet<IModInputCombination>();
-        private readonly float[] _timeout = new float[MaxUsefulKey];
-        private readonly int[] _gameBindingCounter = new int[MaxUsefulKey];
+        private readonly float[] _timeout = new float[ModInputLibrary.MaxUsefulKey];
+        private readonly int[] _gameBindingCounter = new int[ModInputLibrary.MaxUsefulKey];
         private IModInputCombination _currentCombination;
         private int _lastSingleUpdate;
         private int _lastCombinationUpdate;
         private readonly IModLogger _logger;
         private readonly IModConsole _console;
 
+        public IModInputTextures Textures { get; }
+
         public ModInputHandler(IModLogger logger, IModConsole console, IHarmonyHelper patcher, IOwmlConfig owmlConfig, IModEvents events)
         {
+            var textures = new ModInputTextures();
+            textures.FillTextureLibrary();
+            Textures = textures;
```

Comment:
You could save a line:
```
Textures = new ModInputTextures();
Textures.FillTextureLibrary();
```

Reply:
FillTecctureLibrary is internal, I preferred that to cast